src/app/components/game/game.component.ts

```diff
@@ -298,6 +342,8 @@ export class GameComponent implements OnInit, OnChanges {
         // this.game = new Game('1k6/1p6/8/2P5/5p2/4P3/1K6/8 w - - 0 1');
         this.CURSOR_DATA = {
             mouseIsDown: false,
+            mouseLeftIsDown: false,
+            mouseRightIsDown: false,
```

Comment:
We don't want recurring values, it makes us reassign things like fools too much.

This (https://github.com/JBader89/schackmatt/pull/1) is more clear.  Let me know if this works for you.